CMakeLists.txt

```diff
@@ -45,16 +45,13 @@ if (STATIC_BUILD)
             set(CMAKE_CXX_FLAGS_RELEASE "/MT")
             set(CMAKE_CXX_FLAGS_DEBUG "/MTd")
         endif()
-        if (BUILD_PYTHON)
-            add_definitions(-DBOOST_PYTHON_STATIC_LIB)
-        endif()
     else()
         set(CMAKE_FIND_LIBRARY_SUFFIXES ".a" ".so")
         set(link_param "-static")
-        if (BUILD_PYTHON)
-            find_package(ZLIB)
-            find_package(EXPAT)
-            find_package(Threads)
+        if (BUILD_PYTHON)	
```

Comment:
nitpick - looks like whitespace at the end of the lines has been introduced here

Reply:
Fixed now